### PR TITLE
Metalcraft: crown, ring, and bracelet

### PR DESCRIPTION
**Pull request by ituluwituluwzev**
Forging a crown is suggested here: http://smf.cataclysmdda.com/index.php?topic=5865.msg307755#msg307755.

In jewelry shop and other places, we can find a gallery book with which one may try to replicate ancient crown and various fancy trinkets with gold and silver salvaged from bank vault. In museum, we can occasionally find a Bactrian crown that looks even fancier.

Be it a king, a queen, or an extradimensional prince of blob, who wears the crown is the post-apocalyptic political detail that I leave to the player's decision.

### CODE REVIEW

**Comment by ituluwituluwzev** [comment]
Terminated: data/mods//dda/../../json/items/armor.json: line 1963:3: member not found: name

Does this mean that two items cannot have the same "name"?

File: data/json/recipes/armor/head.json [hunk 1]
```diff
@@ -92,6 +92,19 @@
...
+    "result": "crown_golden_survivor",
+    "type": "recipe",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_HEAD",
+    "skill_used": "fabrication",
+    "difficulty": 6,
+    "time": 600000,
+    "book_learn": [ [ "jewelry_book", 6 ] ],
+    "qualities": [ { "id": "HAMMER", "level": 3 } ],
+    "tools": [ [ [ "surface_heat", 10, "LIST" ] ] ],
```
Comment (Night-Pryanik) [inline data/json/recipes/armor/head.json]: `surface_heat` group include `charcoal smoker` and `fire nearby` tools. They don't fit as metalworking tools here. Better explicitly  replace them by `soldering iron` and `integrated toolset`.
The same for ring crafting below.

**Comment by Night-Pryanik** [comment]
AFAIK, gold is heated by the burner. Gold is melted at more than 1000 C, common soldering iron can't achieve this temperature.

**Comment by ituluwituluwzev** [comment]
I'll add a proper forge to the recipe.

**Comment by ituluwituluwzev** [comment]
What is the problem? Do I use copy-from incorrectly?

File: data/json/recipes/armor/other.json [hunk 2]
```diff
@@ -13,6 +13,45 @@
...
+    ],
+    "tools": [
+      [ [ "tongs", -1 ] ],
+      [ [ "anvil", -1 ] ],
+      [ [ "swage", -1 ] ],
+      [ [ "forge", 50 ], [ "oxy_torch", 10 ] ]
+    ],
+    "components": [ [ [ "gold_small", 3 ] ] ]
+  },
+  {
```
Comment (Night-Pryanik) [inline data/json/recipes/armor/other.json]: There is an excessive space after the bracket.
You should run your files through the linting tool (http://dev.narc.ro/cataclysm/format.html).

Comment (ituluwituluwzev) [reply data/json/recipes/armor/other.json]: Thanks!

**Comment by ituluwituluwzev** [comment]
They all go through the web linter now, but some still don't pass the check.

**Comment by Coolthulhu** [comment]
You have to specify the name for each object.
The name should not be identical for different objects.

**Comment by ituluwituluwzev** [comment]
It says "All checks have passed." But some commits remain red. Are there problems yet to be fixed?

**Comment by Broken-Admin** [comment]
I find this to be better implemented as a mod.

**Comment by ituluwituluwzev** [comment]
Making adornment out of shiny metal is something that would happen even when cataclysm took place... It is quite generic.

**Pull request merged**